LibraBFT/Impl/Properties/Util.agda

```diff
@@ -25,6 +25,13 @@ open import LibraBFT.Abstract.Types.EpochConfig UID NodeId
 
 module LibraBFT.Impl.Properties.Util where
 
+module Meta where
+  getLastVoteEpoch : RoundManager → Epoch
+  getLastVoteEpoch = maybe{B = const Epoch} (_^∙ vEpoch) 1 ∘ (_^∙ lSafetyData ∙ sdLastVote)
```

Comment:
> Alternatively, we could just make this the current epoch of the RoundManager, which would make more sense.

Excellent suggestion. I have changed it so that the default is the epoch stored in the round manager's safety data.